src/System.Private.CoreLib/src/System/Diagnostics/SymbolStore/ISymWriter.cs

```diff
@@ -26,7 +24,7 @@ internal interface ISymbolWriter
         // Define a source document. Guid's will be provided for the
         // languages, vendors, and document types that we currently know
         // about.
-        ISymbolDocumentWriter DefineDocument(string url,
+        ISymbolDocumentWriter? DefineDocument(string url,
```

Comment:
Implementations of this return null?

Reply:
yes here https://github.com/dotnet/coreclr/blob/master/src/System.Private.CoreLib/src/System/Reflection/Emit/ISymWrapperCore.cs#L182